cs-minibot-platform-src/src/main/java/simulator/physics/PhysicalObject.java

```diff
@@ -19,14 +19,16 @@
     private Body body;
     private int id;
     private World world;
-    private int size;
+    private float size;
 
+    //this constructor is for bots, no size is passed in, the default bot size is 0.15m, or 6 inches
     public PhysicalObject(String name, int id, World world, float xSpeed,
                           float ySpeed, float xPos, float yPos, int angle,
                           boolean isDynamic) {
         this.name = name;
         this.id = id;
-        this.size = 0;
+        float bot_size = 0.15f;
```

Comment:
since this is constant, do
public static final float BOT_SIZE = 0.15f;

outside of this constructor. refer to it using that.